nucypher/blockchain/eth/interfaces.py

```diff
@@ -796,7 +842,8 @@ class BlockchainInterfaceFactory:
 
     CachedInterface = collections.namedtuple('CachedInterface', ['interface',    # type: BlockchainInterface
                                                                  'sync',
-                                                                 'emitter'])
+                                                                 'emitter',
+                                                                 'gas_strategy'])
```

Comment:
Why do we need this? There's already `interface.gas_strategy`. Perhaps there's something I'm missing.

Reply:
:heavy_check_mark: